test/elixir_search_extractor_web/features/settings_page/settings_page_test.exs

```diff
@@ -0,0 +1,37 @@
+defmodule ElixirSearchExtractorWeb.SettingsPage.SettingsPageTest do
+  use ElixirSearchExtractorWeb.FeatureCase
```

Comment:
```suggestion
  use ElixirSearchExtractorWeb.FeatureCase, async: true
```

Reply:
Added in [6d138e0](https://github.com/rafayet-monon/elixir_search_extractor/pull/27/commits/6d138e0797e370ce45a98cf9233ad84d6400653b)